tests/suite/scale_test.go

```diff
@@ -0,0 +1,260 @@
+package suite
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strconv"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	core "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+
+	"github.com/nginxinc/nginx-gateway-fabric/tests/framework"
+)
+
+var _ = Describe("Scale test", Ordered, Label("nfr", "scale"), func() {
+	promManifest := []string{
+		"scale/prom.yaml",
+	}
+	//matchesManifest := []string{
+	//	"scale/matches.yaml",
+	//}
+	//upstreamsManifest := []string{
+	//	"scale/upstreams.yaml",
+	//}
+	ns := &core.Namespace{
+		ObjectMeta: metav1.ObjectMeta{
+			Name: "scale",
+		},
+	}
+
+	var resultsDir string
+	var outFile *os.File
+	var ngfPodName string
+	var err error
+
+	BeforeAll(func() {
+		// Need longer create timeout to allow for all the apps
+		resourceManager.TimeoutConfig.CreateTimeout = 180 * time.Second
+		Expect(resourceManager.Apply([]client.Object{ns})).To(Succeed())
+		Expect(resourceManager.ApplyFromFiles(promManifest, ngfNamespace)).To(Succeed())
+		resultsDir, err = framework.CreateResultsDir("scale", version)
+		Expect(err).ToNot(HaveOccurred())
+
+		filename := filepath.Join(resultsDir, fmt.Sprintf("%s.md", version))
```

Comment:
This filename formatting is slightly changed now in other tests to account for N+